source/building-and-editing.html.erb

```diff
@@ -447,6 +447,11 @@ order: 3
           <td>Date</td>
           <td>Sets the latest data that will be accepted</td>
         </tr>
+        <tr>
+        <td>Specific answer format (regex)</td>
+        <td>Text<br>Number</td>
```

Comment:
I think we descoped number and it's only text input?

Reply:
Thanks Steven, Sijy has just made the same point. (How did you spot this so quickly!?)

It looks like we added it to textarea though, which seems both pointless and effort we could have spent on number. Oh well! Just updating now.